lms/djangoapps/experiments/tests/test_views.py

```diff
@@ -260,6 +261,11 @@ def _cross_domain_post(self, csrf_token, data, referer=CROSS_DOMAIN_REFERER):
             **kwargs
         )
 
+    def test_white_list_contents_with_cors_header_version(self, *args):  # pylint: disable=unused-argument
```

Comment:
This test will verify the version of `django_cors_headers` and `CORS_ORIGIN_WHITELIST` contents.